config/features.yml

```diff
@@ -358,7 +358,12 @@ features:
     actor_type: user
     enable_in_development: true
     description: >
-      Enhances Find A Forms MVP and enables users to navigate to Form Detail landing pages.
+      Enhances Find A Forms MVP.
```

Comment:
Did you mean to edit/update the description for the "find_forms_mvp_enhancement" feature toggle in the PR?

Reply:
Yes, I figured it would be appropriate to update the flag description since that bulk of work is done, but we are still enhancing the feature.